test_utils/factories.py

```diff
@@ -46,6 +46,7 @@ class Meta(object):
     name = factory.LazyAttribute(lambda x: FAKER.company())
     active = True
     site = factory.SubFactory(SiteFactory)
+    identity_provider = factory.LazyAttribute(lambda x: FAKER.slug())
```

Comment:
I know we are using the `fuzzy` library in other code projects -- does it make sense to use it for `enterprise` as well?